.dependabot/config.yml

```diff
@@ -0,0 +1,28 @@
+version: 1
+
+update_configs:
+  - package_manager: "ruby:bundler"
+    directory: "/"
+    update_schedule: "live"
+    default_reviewers:
+    - tomas-stefano
+    - brenetic
+    - mattempty
+    - njseeto
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "live"
+    default_reviewers:
```

Comment:
I'm missing form this list, but again happy to have less dependabot notifications 😂